strings/LongestRepeatingSequence

```diff
@@ -0,0 +1,28 @@
+public class LongestRepeatingSequence {  
+    //Checks for the largest common prefix  
+    public static String lcp(String s, String t){  
+        int n = Math.min(s.length(),t.length());  
+        for(int i = 0; i < n; i++){  
+            if(s.charAt(i) != t.charAt(i)){  
+                return s.substring(0,i);  
+            }  
+        }  
+        return s.substring(0,n);  
+    }  
+  
+    public static void main(String[] args) {  
+        
+        Scanner sc = new Scanner(System.in);
+        String str = sc.next();  
+        String lrs="";  
+        int n = str.length();  
+        for(int i = 0; i < n; i++){  
+            for(int j = i+1; j < n; j++){  
+                
+                String x = lcp(str.substring(i,n),str.substring(j,n));  
+                if(x.length() > lrs.length()) lrs=x;  
+            }  
+        }  
+        System.out.println("Longest repeating sequence: "+lrs);  
+    }  
+}  
```

Comment:
Hi. Whitespace patterns in the code are unbalanced. Please keep the number of spaces between operators and tokens.
```suggestion
public class LongestRepeatingSequence {
    //Checks for the largest common prefix
    public static String lcp(String s, String t) {
        int n = Math.min(s.length(), t.length());
        for (int i = 0; i < n; i++) {
            if (s.charAt(i) != t.charAt(i)) {
                return s.substring(0, i);
            }  
        }  
        return s.substring(0,n);  
    }  
  
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.next();
        String lrs="";
        int n = str.length();
        for (int i = 0; i < n; i++) {
            for (int j = i + 1; j < n; j++) {
                String x = lcp(str.substring(i, n), str.substring(j, n));
                if(x.length() > lrs.length()) {
                    lrs = x;
                }
            }
        }
        System.out.println("Longest repeating sequence: " + lrs);
    }
}
```

Reply:
Please accept all my pull request, so that I can participate in Hacktoberfest